addon/components/frost-table-body.js

```diff
@@ -51,10 +51,42 @@ export default Component.extend({
 
   // == Functions =============================================================
 
+  setCellWidths (position) {
+    const curBodyColumn = this.$(`.frost-table-row .frost-table-body-cell:nth-child(${position})`)
+
+    // Get width of widest body cell in the column
+    const cellWidths = curBodyColumn.toArray().map((col) => {
+      return this.$(col).outerWidth(true)
+    })
+    const width = Math.max.apply(null, cellWidths)
+
+    curBodyColumn.css('flex', `1 0 ${width}px`)
+
+    return width
+  },
+
+  accountForSelectionColumn (num) {
```

Comment:
Hmm, I'm starting to see repetition here. Also, `isSelectable` is not consistent with `_isSelectable`. Maybe it's time to consider refactoring this out into a helper or mixin.